pkg/api/platformapi/allocatorapi/vacate.go

```diff
@@ -473,14 +477,16 @@ func CheckVacateFailures(failures *models.MoveClustersDetails, filter []string)
 		var ferr error
 		if len(failure.Errors) > 0 {
 			var err = failure.Errors[0]
-			ferr = fmt.Errorf("code: %s, message: %s", *err.Code, *err.Message)
+			ferr = fmt.Errorf("%s (%s)", *err.Message, *err.Code)
```

Comment:
@Z3r0Sum Again does this change breaks our error parsing in soteria?

Reply:
Yes, this change could cause some issues [here](https://github.com/elastic/soteria/blob/master/app/allocator/vacate.go#L319-L327)